### PR TITLE
Catalogs

### PR DESCRIPTION
**Pull request by SunilSimha**
Created this branch off `BPT` so I thought it should merge to that before adding onto master.

Adds the `Pan-STARRS_Survey` class to the list of supported surveys. Can obtain:
* a photometric catalog from a cone search
* a color image (with the user's choice of three bands) cutout.
* a FITS image of the target in the desired band

### CODE REVIEW

**Comment by profxj** [comment]
Will wait for BPT to clear before merging.
And I am pulling in its latest changes now.

File: frb/tests/test_frbsurveys.py [hunk 1]
```diff
@@ -97,3 +99,29 @@ def test_first():
     assert isinstance(first_tbl, Table)
     assert len(first_tbl) == 1
 
+def test_panstarrs():
```
Comment (profxj) [inline frb/tests/test_frbsurveys.py]: For some reason this is failing Travis. 
Probably it won't execute the HTML pull.
Not sure.  But please modify until it does pass
(even if you need to kludge it).

Thanks.

Comment (SunilSimha) [reply frb/tests/test_frbsurveys.py]: It was a bug in my code. It should pass now.

**Comment by profxj** [comment]
@SunilSimha -- Can you close this PR and reissue to master?
I'll then merge.  Thanks.

**Pull request closed without merging**